dotCMS/src/main/webapp/html/portlet/ext/contentlet/view_contentlets.jsp

```diff
@@ -804,7 +804,7 @@
         <!-- END Right Column -->
 
         <!-- START Show Query -->
-        <div id="queryDiv" dojoType="dijit.Dialog" class="content-search__show-query-dialog" style="display: none;padding-top:15px\9;">
+        <div id="queryDiv" dojoType="dijit.Dialog" class="content-search__show-query-dialog noDijitDialogTitleBar" style="display: none;padding-top:15px\9;">

```

Comment:
The `noDijitDialogTitleBar` is a class that dojo (or dijit) add when there is a title bar in the dialog component. In this case I think we can do this by adding a `title=""` to the dialog so it shows the title bar and the close button.

You can always search the dijit documentation and see what properties the widgets takes, for example here is for the dialog: https://dojotoolkit.org/reference-guide/1.6/dijit/Dialog.html